.fixtures.yml

```diff
@@ -2,7 +2,9 @@ fixtures:
   repositories:
     stdlib:        "https://github.com/puppetlabs/puppetlabs-stdlib.git"
     extlib:        "https://github.com/voxpupuli/puppet-extlib"
-    apache:        "https://github.com/puppetlabs/puppetlabs-apache.git"
+    apache:
+      repo: 'https://github.com/puppetlabs/puppetlabs-apache.git'
+      ref: 'v7.0.0'
```

Comment:
The fix was merged. I think you can revert this again.

Reply:
Reverted, TY